docs/markdown/Getting-meson.md

```diff
@@ -24,19 +24,21 @@ several platforms.
 ## Installing Meson with pip
 
 Meson is available in the [Python Package Index] and can be installed
-with `sudo pip3 install meson` which requires root and will install it
-system-wide.
-
-If you have downloaded a copy of the Meson sources already, you can
-install it with `sudo pip3 install path/to/source/root/`.
```

Comment:
The description on installing from a source checkout got dropped entirely.

Reply:
That is intentional. This is a beginner document and that is an advanced concept. Simpler is better.